src/raster/r.timeofconcentration/r.timeofconcentration.html

```diff
@@ -0,0 +1,106 @@
+When <code>vertical_units=feet</code>, elevation drops are multiplied by <code>1/3.28084</code> to
+convert them to meters. When <code>vertical_units=factor</code>, the module multiplies
+elevation drops by the user-supplied factor.</p>
+<p><strong>factor:</strong> Required only when <code>vertical_units=factor</code>. It must convert the
+user’s vertical unit to meters (<code>factor × &lt;your vertical unit&gt; = meters</code>), and
+the user is responsible for providing the correct factor.</p>
+
+<h3>Outputs</h3>
+<p><strong>output:</strong> A raster map of time of concentration per cell [h], computed using
+the Kirpich equation:</p>
+<pre><code>$$
+T_c \;=\; \frac{K \cdot L^a \cdot S_{\text{avg}}^b}{60}
+$$
+</code></pre>
+<p>where <code>L</code> is upstream flow-path length [m], <code>S_{\text{avg}}</code> is path-average
+slope [unitless], <code>K = 0.01947</code>, <code>a = 0.77</code>, <code>b = -0.385</code> are Kirpich
+constants, and the result is converted from minutes to hours by dividing by 60.
+Tc is NULL if <code>L &lt; length_min</code> or outlets are undefined.</p>
+<p><strong>length:</strong> Optional output raster of longest upstream flow-path length per
+cell [m], derived from <code>r.stream.distance</code>.</p>
+<p><strong>drop:</strong> Optional output raster of flow-path elevation drop per cell [m]
+(&ge; 0), computed as the maximum elevation difference.</p>
+<p><strong>sbar:</strong> Optional raster of path-average slope per cell [unitless], calculated
+as <code>S_{\text{avg}} = \max(\frac{\Delta z}{L}, \text{slope}_\text{min})</code>,
+where <code>\Delta z</code> is the drop and <code>L</code> is the length.</p>
+
+<h2>Notes</h2>
+<ul>
+  <li>CRS-aware distance <code>r.stream.distance</code> returns lengths in meters,
+    even if the CRS is geographic or projected (meters or feet).</li>
+  <li>Static Tc Tc is a steady-state estimate; no temporal
+    routing or storage is modeled.</li>
+</ul>
+
+<h2>EXAMPLE</h2>
+<p>These examples use the North Carolina sample dataset.</p>
+<p>Calculate time of concentration using <code>r.watershed</code> and <code>r.timeofconcentration</code>:</p>
+<pre><code class="language-sh"># set the region
+g.region -p raster=elevation
+
+# calculate positive flow accumulation and drainage directions using r.watershed
+r.watershed -sa elevation=elevation drainage=fdr stream=str threshold=10
+
+# compute the time of concentration
+r.timeofconcentration elevation=elevation direction=fdr streams=str tc=tc_nc
+
+# use length_min parameter for coarser tc on important streams only
+r.timeofconcentration elevation=elevation direction=fdr streams=str tc=tc_nc_250 length_min=250
+
+# if the vertical units of the DEM are in feet
+r.timeofconcentration elevation=elevation vunits=feet dir=fdr str=str tc=tc
+
+# if the vertical units of the DEM are in units other than meters or feet (e.g., cm)
+r.timeofconcentration elevation=elevation vunits=factor factor=0.1 dir=fdr str=str tc=tc
```

Comment:
To fit with the comment:

```suggestion
r.timeofconcentration elevation=elevation vunits=factor factor=0.01 dir=fdr str=str tc=tc
```